ApplicationLibCode/ProjectDataModel/WellLog/RimWellLogTrack.cpp

```diff
@@ -1263,6 +1267,48 @@ bool RimWellLogTrack::isEmptyVisiblePropertyRange() const
            1.0e-6 * std::max( 1.0, std::max( m_visiblePropertyValueRangeMax(), m_visiblePropertyValueRangeMin() ) );
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimWellLogTrack::updateDepthAxisVisibility( RimDepthTrackPlot::DepthOrientation orientation,
+                                                 bool                                isFirstTrack,
+                                                 bool                                isLastTrack )
```

Comment:
The method updates visibility of all axes, not just depth. Maybe updateAxesVisibility is more correct?